.github/scripts/version_updater.sh

```diff
@@ -67,7 +75,16 @@ if [[ -z "${role}" ]]; then
 fi
 
 # Get latest version.
-version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+if [[ "${type}" == "github" ]]
+then
+  version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+elif [[ "${type}" == "gitlab" ]]
+then
+  version="$(gitlab_api "projects/${source_repo}/releases" | jq '.[0].tag_name' | tr -d '"v')"
+else
+  echo_red 'Unknown source type. Terminating.'
+  exit 128
+fi
```

Comment:
Let's use a case statement here. Looks cleaner than a bunch of ifs.

```suggestion
case "${type}" in
  github)
    version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
    ;;
  gitlab)
    version="$(gitlab_api "projects/${source_repo}/releases" | jq '.[0].tag_name' | tr -d '"v')"
    ;;
  *)
    echo_red 'Unknown source type. Terminating.'
    exit 128
    ;;
esac
```